bundles/org.openhab.binding.deconz/README.md

```diff
@@ -235,6 +239,8 @@ Bridge deconz:deconz:homeserver [ host="192.168.0.10", apikey="ABCDEFGHIJ" ] {
 Switch                  Livingroom_Presence     "Presence Livingroom [%s]"          <motion>        { channel="deconz:presencesensor:homeserver:livingroom-presence:presence" }
 Number:Temperature      Livingroom_Temperature  "Temperature Livingroom [%.1f °C]"  <temperature>   { channel="deconz:temperaturesensor:homeserver:livingroom-temperature:temperature" }
 Number:Dimensionless    Livingroom_Humidity     "Humidity Livingroom [%.1f %%]"     <humidity>      { channel="deconz:humiditysensor:homeserver:livingroom-humidity:humidity" }
+String                  Livingroom_voc_label    "Air quality Livingroom [%s]"       <none>          { channel="deconz:airqualitysensor:homeserver:livingroom-voc:airquality" }
+Number:Dimensionless    Livingroom_voc          "Air quality [%d ppb]"              <none>          {           channel="deconz:airqualitysensor:homeserver:livingroom-voc:airqualityppb" }
```

Comment:
Sorry to bother you again, but now when reading the adapted and formatted README two new comments:
- I think you should remove "<none>", i.e. no category.
- There seem to be some alignment problem with the last line: multiple spaces before "{ channel=".

Reply:
I fixed the indentation in the layout.
And I removed the None category from the example.